Updating the Book dash chapter in community handbook (#2062)
alan-turing-institute/the-turing-way: book/website/community-handbook/bookdash/bookdash-selection.md

(ch-bookdash-preparation)=
# Participant Selection Process

> This subchapter explains the process of selection and logistics for organising the book dash.

Author: ```suggestion
This subchapter explains the process of selection and logistics for organising the Book Dash.
```

## Selection of Book Dash Planning Committee

To ensure that the Book Dash continues to evolve as per the community needs and preferences, invite returning members of the Book Dash events to join a planning committee.

> A member of the Book Dash planning committee participates in the planning meetings, provides feedback on the ongoing process and when possible takes the lead on tasks organisation related to The Book Dash.

Author: ```suggestion
Members of the Book Dash planning committee participate in the planning meetings, provides feedback on the ongoing process and when possible takes the lead on organisation tasks related to the Book Dash.
```

- **Debrief meeting**: attend a meeting after the event to discuss what went well and what we can improve going forward (time commitment: 1 hour)
- **Shared Documents**: set up or review shared notes, application form, review rubrics or other required documents for the Book Dash (time commitment: relative to the type of document)
- **Post-event Documents**: ensure the update of {ref}`contributors-record`, help document the event report and update the Book Dash chapter online 

The invitation to join the planning committee is sent by the core team members based on the applications sent by returning members of the Book Dash expressing their interest in mentoring.
This is a volunteer position and there are no expectations of commitment to do multiple tasks described above.

## Selection of Book Dash Participants

The Book Dash Planning Committee meets after they have individually scored the applications using the rubrics defined in the previous subchapter.

> They agree upon a final set of applicants selected to participate in the book dash.

Author: ```suggestion
They agree upon a final set of applicants selected to participate in the Book Dash.
```

The panel members also create constructive feedback on the application that can be shared with the applicants who were not selected so that they can assess other pathways to engage with the project and core team.

> All applicants are contacted at least 4 weeks in advance to ensure that they can assess their situation and availability for the book dash.

Author: ```suggestion
All applicants are contacted at least 4 weeks in advance to ensure that they can assess their situation and availability for the Book Dash.
```

> Upon confirmation, they are provided with registration details through which they can share any information that can help them and the organisers prepare for the book dash.

Author: ```suggestion
Upon confirmation, they are provided with registration details through which they can share any information that can help them and the organisers prepare for the Book Dash.
```

> Participants are asked to get familiarised with [_The Turing Way_ GitHub repository](https://github.com/alan-turing-institute/the-turing-way) where they will make their contributions during the book dash.

Author: ```suggestion
Participants are asked to get familiarised with [_The Turing Way_ GitHub repository](https://github.com/alan-turing-institute/the-turing-way) where they will make their contributions during the Book Dash.
```

> The project's [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) and {ref}`Code of Conduct<ch-coc>` are also shared with the participants that are applicable to the book dash as well.

Author: ```suggestion
The project's [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) and {ref}`Code of Conduct<ch-coc>` are also shared with the participants that are applicable to the Book Dash as well.
```
